tests/e2e/Services/Users/UsersBase.php

```diff
@@ -1553,6 +1553,137 @@ public function testDeleteUser(array $data): array
         return $data;
     }
 
+    public function testUsetJWT()
```

Comment:
Is the function name supposed to be:

```suggestion
    public function testUserJWT()
```